docs/source/api/image.rst

```diff
@@ -49,7 +49,7 @@ Enumerations
 Image Classes
 -------------
 
-Class Hierachy
+Class Hierarchy
 ^^^^^^^^^^^^^^
```

Comment:
I missed this earlier, reST specifies that the underline (^^^...) be at least, as long as the section title.

Due to this, the docs are failing to build.